app/components/check_your_answers_summary/component.rb

```diff
@@ -116,9 +116,13 @@ def file_links_for(document, translations)
       scope
         .order(:created_at)
         .select { |upload| upload.attachment.present? }
-        .map { |upload| link_to(upload.name, upload.url) }
+        .map { |upload| uploaded_document_url(upload) }
         .join(", ")
         .html_safe
     end
+
+    def uploaded_document_url(upload)
```

Comment:
```suggestion
    def uploaded_document_link(upload)
```

I'd suggest naming this `link` as it's returning more than just a URL.